README.md

```diff
@@ -84,14 +85,20 @@ SymPy is a pure Python library for symbolic mathematics. It aims to become a ful
        Recursing through an Expression Tree
 
 
-The tutorial will only assume a basic knowledge of Python. No prior knowledge of SymPy or other Python libraries is required, although it is suggested that attendees be familiar with the IPython notebook. A mathematical knowledge of calculus is recommended.
+The tutorial will only assume a basic knowledge of Python and C++. No prior knowledge of SymPy or other Python libraries is required, although it is suggested that attendees be familiar with the IPython notebook. A mathematical knowledge of calculus is recommended.
 
 We recommend that the attendees install the Anaconda Python distribution which includes SymPy, NumPy, and IPython. Once Anaconda is installed simply type the following in a terminal to install the necessary packages:
 
 `$ conda install numpy ipython-notebook sympy`
 
 Other alternative installation instructions can be found here: http://docs.sympy.org/dev/install.html
 
+SymEngine and its Python wrapper can be installed directly through:
+
+`$ conda install -c conda-forge python-symengine`
+
+Other alternative installation instructions can be found at: https://github.com/symengine/symengine/wiki/Building-SymEngine and https://github.com/symengine/symengine.py/
+
```

Comment:
@ShikharJ @ranjithkumar007 What about other wrappers (Ruby, Julia, Haskell) ? I think it will be good if we talk basics of other wrappers as well.